### PR TITLE
Allow to login to OCM with a token

### PR DESCRIPTION
**Pull request by apodhrad**
Signed-off-by: Andrej Podhradsky <apodhrad@redhat.com>

### CODE REVIEW

File: ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@ CLUSTER_NAME=${CLUSTER_NAME:-$NAMESPACE}
 CLUSTER_VERSION=${CLUSTER_VERSION:-}
 SSO_CLIENT_ID=$(cat "${CLUSTER_PROFILE_DIR}/sso-client-id")
```
Comment (jupierce) [inline ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh:11]: Do you need to check whether these files exist now before cat'ing them?

Comment (apodhrad) [reply ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh]: Yes. I have fixed that

File: ci-operator/config/openshift/dptp-workflow-test/openshift-dptp-workflow-test-master.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ tests:
     cluster_profile: osd-ephemeral
     env:
       CLUSTER_DURATION: "300"
-      CLUSTER_VERSION: 4.6.12
+      CLUSTER_VERSION: 4.10.3
```
Comment (bradmwilliams) [inline ci-operator/config/openshift/dptp-workflow-test/openshift-dptp-workflow-test-master.yaml:33]: I would say if you're going to bump the CLUSTER_VERSION, then you should also bump the other `4.6` values up to `4.10` as well.

**Review by bradmwilliams — CHANGES_REQUESTED** [review]

**Comment by bradmwilliams** [comment]
Unfortunately, your failed tests seem to be related to the ongoing issues that they are having on the `build02` cluster.

**Comment by apodhrad** [comment]
Hi @bradmwilliams is the problem with `build02` being tracked somewhere?

File: ci-operator/config/openshift/dptp-workflow-test/openshift-dptp-workflow-test-master.yaml (hunk 2 above)
Comment (apodhrad) [reply ci-operator/config/openshift/dptp-workflow-test/openshift-dptp-workflow-test-master.yaml:33]: When I tried it previously (to update all occurrences of 4.6 to 4.10), the check `
ci/prow/prow-config-filenames` failed.
But now, I have tried again and the same check passed. So, I will keep it.

**Comment by apodhrad** [comment]
Hi @bradmwilliams @jupierce OSD clusters are provisioned correctly but one test is still failing
ci/rehearse/periodic-ci-openshift-knative-serverless-operator-main-4.9-osd-e2e-osd-49-continuous

I think that changes in this PR did not break the above test.
But to be sure I have also contacted one of the authors of that test - @mgencur

**Comment by mgencur** [comment]
@apodhrad Hey! I've looked at the failure and it seems to be a problem on our side in the given repository. Not related to this PR.

**Comment by bradmwilliams** [comment]
> Hi @bradmwilliams is the problem with `build02` being tracked somewhere?

There is a channel in slack: #incident-unbalanced-registry-load

**Review by bradmwilliams — APPROVED** [review]

**Comment by jupierce** [comment]
@apodhrad if you would like to add yourself as an approver to the OWNERS file, I will help you get that merged.

**Pull request merged**